wire-runtime-swift/src/test/swift/sample/Dinosaur.swift

```diff
@@ -8,13 +8,17 @@ public struct Dinosaur {
     /**
      * Common name of this dinosaur, like "Stegosaurus".
      */
+    @Defaulted(defaultValue: "")
```

Comment:
- [ ] Should _probably_ update the README to include info about @Defaulted and this updated model

Reply:
yeap, thanks for reminding

Reply:
done